### PR TITLE
Upgrade kotlin and kotlin-coroutines to versions 1.3.72 and 1.3.9

### LINKED ISSUE #378: Upgrade kotlin coroutines to 1.3.5 blocked by deprecated publish
See https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-reactive/kotlinx.coroutines.reactive/kotlinx.coroutines.-coroutine-scope/publish.html

### PR DESCRIPTION
**Pull request by brannstrom**
Fixes #378

## Checklist

- [x] Pull requests follows the [contribution guide](https://github.com/graphql-java-kickstart/graphql-java-tools/wiki/Contribution-guide)
- [x] New or modified functionality is covered by tests

## Description
Replaced deprecated CoroutineScope.publish(...) with top-level publish(...) method in SchemaParserOptions(See: https://github.com/Kotlin/kotlinx.coroutines/blob/master/reactive/kotlinx-coroutines-reactive/src/Publish.kt). No new tests were added but existing ones looked sufficient to me. I noticed that @oliemansm didn't upgrade the kotlin-coroutines lib in the "upgrade-dependencies" branch (See: commit 4543ccd) so there might be a reason to not do this update that I am not aware of?!

### CODE REVIEW

File: src/test/groovy/graphql/kickstart/tools/SchemaParserSpec.groovy
Comment (vojtapol) [inline src/test/groovy/graphql/kickstart/tools/SchemaParserSpec.groovy:269]: Please match the formatting in the other multi-line String. Should be indented by 4 spaces.

**Review by vojtapol — CHANGES_REQUESTED** [review]
This looks good. I would be happy to merge once we get 2 more approvals from the community. Especially, from those that actively use coroutines with `graphql-java-tools.

File: src/test/groovy/graphql/kickstart/tools/SchemaParserSpec.groovy
Comment (brannstrom) [reply src/test/groovy/graphql/kickstart/tools/SchemaParserSpec.groovy]: This test was failing for me while building locally, that is why I changed the formatting. I've commited a new version where I use stripMargin() instead.

**Comment by brannstrom** [comment]
@vojtapol Hi, is it possible to get any progress with this PR?

**Review by vojtapol — APPROVED** [review]
Thank you for the contribution. Note that it may take some time for a new version of the library to be released.

**Pull request merged**
